schematic/manifest/generator.py

```diff
@@ -1677,16 +1682,25 @@ def create_manifests(
                     "Please check your submission and try again."
                 )
 
-        data_model_parser = DataModelParser(path_to_data_model=path_to_data_model)
+        if not graph_data_model:
+            if data_model_graph_pickle:
+                with open(data_model_graph_pickle, "rb") as f:
```

Comment:
Is this file always guaranteed to fit into memory? Really large files are likely to cause OOM issues here. Likely you are not dealing with large amounts of data, but thought I'd throw the question out there.

Reply:
I think the largest of these files will be <100MB so I don't think we need to worry about it for this release.